utils/bluez-tools/Makefile

```diff
@@ -0,0 +1,54 @@
+#
+# Copyright (C) 2021 Karl Osterseher <karli_o@gmx.at>
+#
+# This is free software, licensed under the GNU General Public License v2.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=bluez-tools
+PKG_VERSION:=20201025.f653217
+PKG_RELEASE:=$(PKG_SOURCE_VERSION)
```

Comment:
Sorry but I still don't agree.
This should be `PKG_RELEASE:=1` and should be update if you changes the Makefile or add a patch to change the source.
You could also use `PKG_RELEASE:=$(AUTORELEASE)` Then the commit history is used to check the last **PKG_VERSION** update, and increments the number for each commit automaticaly.